docs/docs/recipes.md

```diff
@@ -853,6 +853,113 @@ export const pageQuery = graphql`
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing data from Contentful
+
+#### Prerequisites
+
+- A [Contentful account](https://www.contentful.com/)
+- A [Contentful space](https://www.contentful.com/faq/basics/#how-do-i-create-a-space)
+- One [Content model](https://www.contentful.com/developers/docs/concepts/data-model/)
+- One or more pieces of published content to [avoid build errors](https://github.com/gatsbyjs/gatsby/issues/15344#issuecomment-520151776)
+- The [space ID and content delivery API key](https://www.contentful.com/developers/docs/concepts/apis/#content-delivery-api) for your space
+
+The screenshots and examples in this recipe were set up with a new Contentful space with a "News" content model that includes "Title" and "Body" fields.
+
+#### Directions
+
+1. Install the `gatsby-source-contentful` plugin in your Gatsby site:
+
+```shell
+npm install --save gatsby-source-contentful
+```
+
+2. Add the following plugin entry to your `gatsby-config.js` file. Use your own space ID and access token from the API keys section of your Contentful space. You should consider using [environment variables](https://www.gatsbyjs.org/docs/environment-variables/) to store your space ID and token.
+
+```javascript:title=gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-source-contentful`,
+    options: {
+      spaceId: `your_space_id`, // or process.env.CONTENTFUL_SPACE_ID
+      accessToken: `your_access_token`, // or process.env.CONTENTFUL_TOKEN
+    },
+  },
+]
+```
+
+3. Run `gatsby develop` and make sure the site compiled successfully.
+
+4) Query data with the [GraphiQL editor](/docs/introducing-graphiql/) at `https://localhost:8000/___graphql`. The Contentful plugin adds several new node types to your site, including every content type in your Contentful website. For example, a site with a `News` content type and two fields produces an `allContentfulNews` node type in GraphQL.
+
+![the graphql interface, with a sample query outlined below](./images/recipe-sourcing-contentful-graphql.png)
+
+To query for news articles from Contentful, use the following GraphQL query:
+
+```graphql
+{
+  allContentfulNews {
+    edges {
+      node {
+        body {
+          body
+        }
+        title
+      }
+    }
+  }
+}
+```
+
+Contentful nodes also include several metadata fields like `createdAt` or `node_locale`.
+
+5. To source Contentful data in your Gatsby site, create a page component with a GraphQL query and some code to iterate over news articles from your sample space:
+
+```jsx
+import React from "react"
+import { graphql } from "gatsby"
+
+const NewsPage = ({ data }) => (
+  <div>
+    <h1>News</h1>
+    {data.allContentfulNews.edges.map(({ node }) => (
+      <>
+        <h2>{node.title}</h2>
+        <p>
+          <strong>{node.createdAt}</strong>
+          {" — "}
+          {node.body.body}
```

Comment:
When setting body as a long text node, you get the option to make your text bold, italic, ...
![image](https://user-images.githubusercontent.com/1120926/65020211-04df2780-d92d-11e9-8fef-8e677e9532ad.png)

When just spitting out `body.body` and make a piece of the body bold.
![image](https://user-images.githubusercontent.com/1120926/65020283-2dffb800-d92d-11e9-841c-3ff3e919ca12.png)

Unsure if we need to put this in the recipe, but maybe some extra notes might help.

Reply:
Yes, a prior version of the recipe mentioned that, but the gatsby-source-contentful readme does a pretty good job covering that topic and more as well.

Reply:
I added a markdown transform to the recipe example to address this.